parlai/agents/transformer/modules.py

```diff
@@ -98,9 +112,15 @@ def __init__(self, opt, dictionary):
         if n_positions < 0:
             raise ValueError('n_positions must be positive')
 
+        self.activation = opt.get('activation', 'relu')
```

Comment:
not sure if we need these (activation, variant ... ) since these are passed into `build_encoder` via opt

Reply:
Indeed we do not.